src/programs/raster/mdimtranslate.rs

```diff
@@ -231,15 +231,55 @@ mod tests {
 
     use crate::{DatasetOptions, Driver, GdalOpenFlags};
 
+    /// Create a copy of the test file in a temporary directory
+    /// and returns a tuple of the temp dir (for clean-up) as well as the path to the file.
+    /// We can remove this when <https://github.com/OSGeo/gdal/issues/6253> is resolved.
+    struct TempDataset {
+        _temp_dir: tempfile::TempDir,
+        temp_path: PathBuf,
+    }
+
+    impl TempDataset {
+        pub fn fixture(name: &str) -> Self {
+            let path = std::path::Path::new(file!())
```

Comment:
Is this really better than `Path::new("fixtures")`? The tests should run in the root directory of the project.

Reply:
I can revert that, just copied it from fixtures!